src/window/window.rs

```diff
@@ -4,6 +4,10 @@ use sf_bool_ext::SfBoolExt;
 use system::{Vector2i, Vector2u};
 use window::{ContextSettings, Event, Style, VideoMode};
 
+/// The system native window handle type. Can be used to initialize SFMl within
```

Comment:
`s/SFMl/SFML`

Wait... This doesn't sound right. I mean... We're not initializing SFML as a whole here, only a window, aren't we?

Reply:
Yes. We initialize an SFML window based on an existing system window. I'm not sure of the internals, but I believe SFML auto-initializes other internals on an as-needed basis (e.g. connections to the X window server).

I'm not sure whether SFML takes control of the specified window handle or if it just initializes an OpenGL context so it can draw on it and otherwise leaves it alone. It might depend on whether you try to call other methods like set_title, set_size, etc. I haven't needed those in my use case (using SFML for an xsecurelock screensaver), so I don't know what happens if you call them (though I may experiment with them).